src/app/core/side-menu/notification/invite-notif/invite-notif.component.ts

```diff
@@ -28,31 +30,36 @@ export class InviteNotifComponent implements OnInit {
               private notificationComponent: NotificationComponent) { }
 
   ngOnInit() {
-    this.user = this.notificationComponent.user
+    this.pendingInvitations = cloneDeep(this.notificationComponent.user.pendingInvitations);
   }
 
   accept(projectId) {
+    if (!this.netConnected) {
+      this.showError('Accept or deny is not available in offline mode.');
+      return;
+    }
+
     if (!this.acceptDisabledIndex) {
       this.acceptDisabledIndex = true;
       this.notificationService.accept(projectId).then((project) => {
         project.activities = [];
         this.store.dispatch(this.projectsActions.addProject(project));
-        this.user.pendingInvitations.map((obj, index) => {
+        this.pendingInvitations.map((obj: Project, index: number) => {
           if (obj.id === projectId) {
-            this.user.pendingInvitations.splice(index, 1);
+            this.pendingInvitations.splice(index, 1);
```

Comment:
I prefer these things in reducer. so there is no need to change state in components.